src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -4182,6 +4182,26 @@ public static void Xml_Soap_TypeWithReadOnlyMyCollectionProperty()
         Assert.True(value.Collection.SequenceEqual(actual.Collection));
     }
 
+    [Fact]
+    public static void Xml_XmlTextAttributeTest()
```

Comment:
Does this test pass in Desktop as well or it also tests .Net Core specific behavior? 

You could verify this by doing the following:
`build.cmd -framework:netfx`
`msbuild <path-to-test-csproj> /t:rebuildandtest /p:targetgroup=netfx`